zerver/views/user_settings.py

```diff
@@ -192,9 +192,11 @@ def json_change_notify_settings(request, user_profile,
                                                                      default=None),
                                 enable_digest_emails=REQ(validator=check_bool,
                                                          default=None),
+                                message_content_in_email_notifications=REQ(validator=check_bool,
+                                                                        default=None),
```

Comment:
This line is failing the Travis integrated tests (due to indenting style) - the failing tests will show the exact error.

Reply:
Yeah, got it. Thanks for reviewing :)